src/App.jsx

```diff
@@ -24,16 +24,30 @@ function App() {
       ) : undefined}
       {screen === "game" ? (
         <GameScreen
+          onShowResultScreen={() => {
+            setScreen("result");
+          }}
           onEndQuiz={() => {
             setScreen("welcome");
           }}
+          onGetAmountCorrectAnswers={() => {
```

Comment:
A better name for this event would be `onCorrectAnswer` - that's what triggers the increase of `amountCorrectAnswers`